packages/getting-started/LICENSE

```diff
@@ -0,0 +1,25 @@
+Copyright (c) IBM Corp. 2017. All Rights Reserved.
```

Comment:
2018

Reply:
Technically, this is a migration of existing licensed code; should it change to 2018, or be 2017-2018?